scripts/scripts.js

```diff
@@ -132,15 +133,228 @@ export function decorateMain(main) {
   decorateQuickLinks(main);
 }
 
+function initWebSDK(path) {
+  return new Promise((resolve) => {
+    import(path)
+      .then(() => {
+      })
+      .then(() => {
+        resolve();
+      })
+      .catch(() => {
+      });
+  });
+}
+
+function onDecoratedElement(fn) {
+  // Apply propositions to all already decorated blocks/sections
+  const elements = document.querySelectorAll('[data-block-status="loaded"],[data-section-status="loaded"]');
+  if (elements.length > 0) {
+    fn();
+    elements.forEach((element) => {
+      if (element.classList.contains('visibility-hidden')) {
+        element.classList.remove('visibility-hidden');
+      }
+    });
+  }
+
+  const observer = new MutationObserver((mutations) => {
+    if (mutations.some((m) => m.target.tagName === 'BODY'
```

Comment:
Maybe try simplifying this with filter/map?

Reply:
How so? You need to catch:
- blocks that become visible
- sections that become visible
- anything else added to the body (i.e. outside of sections, blocks like header, footer, cookie banners, etc.)

And the observation config below already filters on attributes for blocks and sections.
What more simplifying would you do here?